src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -48,6 +67,31 @@ export const SueReportLocation = ({
   const zipCodeInputRef = useRef();
   const cityInputRef = useRef();
 
+  const queryVariables = {
+    start_date: '1900-01-01T00:00:00+01:00',
+    status: `${SueStatus.UNPROCESSED},${SueStatus.OPEN},${SueStatus.IN_PROCESS},${SueStatus.WAIT_REQUESTOR},${SueStatus.WAIT_THIRDPARTY}`
+  };
+
+  const { data, isLoading } = useQuery(
+    [QUERY_TYPES.SUE.REQUESTS, queryVariables],
+    () => getQuery(QUERY_TYPES.SUE.REQUESTS)(queryVariables),
+    {
+      cacheTime: 1000 * 60 * 60 * 24 // 24 hours
```

Comment:
can 24 hours be too long? what about 12 hours? or "until the end of the day" calculation?

did you try to send a report, make it visible per backend and create a new report? does the previous report be marked on the map? or is the cache object loaded without the previous entry?

Reply:
I think it would make more sense to do it by the end of the day. 👍 I'll update the code for this.

Reply:
I also tested. If a new report is shared, the newly created report is not displayed in `SueReportLocation`. So it works with cache. But if I close and open the application, I can see it.

https://github.com/smart-village-solutions/smart-village-app-app/pull/1059/commits/dd5effa061a261ed78b21ea6fbcb754f2f024336